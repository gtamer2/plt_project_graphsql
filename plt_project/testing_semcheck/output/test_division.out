SAST: 
sprogram_string: 
(Int : (Int : 8) / (Int : 2));
 
svarmap_string: 
[]
 
sbindmap_string: 
[]