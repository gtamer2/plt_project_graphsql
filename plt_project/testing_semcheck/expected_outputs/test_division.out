Processing all stmts
Initial Expression: (8 / 2) 
Evaluating expression: (8 / 2)
Evaluating expression: 8
Evaluating expression: 2
Result: true
